src/status_im/contexts/wallet/collectible/options/view.cljs

```diff
@@ -0,0 +1,74 @@
+(ns status-im.contexts.wallet.collectible.options.view
+  (:require
+    [quo.core :as quo]
+    [react-native.platform :as platform]
+    [status-im.config :as config]
+    [status-im.constants :as constants]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [utils.url :as url]))
+
+(defn get-url
```

Comment:
think I should move this else where. any suggestions? 🤔 

It's not collectibles specific 👍

Reply:
yeah utils sounds good, but as I mentioned in other comments on prs a generic utils is not great. I prefer if they're categorised to avoid having some random bucket of methods.
Perhaps 
status-im.contexts.wallet.utils.external-links
or
status-im.contexts.wallet.utils.external-sites
🤔